CMakeLists.txt

```
@@ -26,6 +26,11 @@ set(ADDITIONAL_LIB_INCLUDES "")
 # ========= Configuration Options =========
 option(INCLUDE_MPQ_SUPPORT "Enable StormLib and MPQ archive support" OFF)
 option(GBI_UCODE "Specify the GBI ucode version" F3DEX_GBI_2)
+option(ENABLE_EXP_AUTO_CONFIGURE_CONTROLLERS "Enable experimental auto-configure controllers feature (assign each controller to a specific port)" OFF)
```

i'm not sure what the best wording for this would be, but "assign each controller to a specific port" implies functionality that this doesn't have (such as "setting" a controller to always be on a certain port).

---

hmm "Have the possibility to no longer initialise all controller on port 1" ?

---

maybe something like

```suggestion
option(ENABLE_EXP_AUTO_MULTIPLAYER_CONTROLLERS "Experimental: add support for an alternate default controller-to-port routing where the first four SDL controllers use distinct player ports" OFF)
```

---

I have made the modification manually